core/app/c/[communitySlug]/fields/page.tsx

```diff
@@ -59,7 +59,6 @@ export default async function Page(props: Props) {
 
 	const pubFields = await getPubFields({
 		communityId: community?.id as CommunitiesId,
-		includeRelations: true,
```

Comment:
`includeRelations` was originally meant to hide relationship fields while we were still building them out. now that they are being used everywhere (I think) I've removed the param in favor of an `isRelated: boolean` which can filter for only relations (or non-relations)